multinet/db.py

```diff
@@ -129,6 +130,9 @@ def create_workspace(name: str) -> None:
         # Could only happen if there's a name collisison
         raise InternalServerError()
 
+    workspace_mapping.cache_clear()
+    get_workspace_db.cache_clear()
+
```

Comment:
This is the only instance of these lines without a comment

Reply:
Good catch, I'll add it